src/resources/language/_language-nl.yml

```
@@ -0,0 +1,90 @@
+toc-title-document: "Inhoudsopgave"
+toc-title-website: "Op deze pagina"
+
+section-title-abstract: "Samenvatting"
+section-title-appendices: "Bijlagen"
+section-title-footnotes: "Voetnoten"
+section-title-references: "Referenties"
+section-title-reuse: "Hergebruik"
+section-title-citation: "Citaat"
+
+appendix-attribution-cite-as: "Citeer dit werk als:"
+appendix-attribution-bibtex: "BibTeX citaat:"
+
+title-block-author-single: "Auteur"
+title-block-author-plural: "Auteurs"
+title-block-affiliation-single: "Affiliatie"
+title-block-affiliation-plural: "Affiliaties"
+title-block-published: "Publicatiedatum"
+
+callout-tip-caption: "Tip"
+callout-note-caption: "Opmerking"
+callout-warning-caption: "Waarschuwing"
+callout-important-caption: "Belangrijk"
+callout-caution-caption: "Voorzichtigheid"
+
+code-summary: "Code"
+
+code-tools-menu-caption: "Code"
+code-tools-show-all-code: "Alle code tonen"
+code-tools-hide-all-code: "Alle code verbergen"
+code-tools-view-source: "Broncode bekijken"
+code-tools-source-code: "Broncode"
+
+copy-button-tooltip: "Kopieer naar klembord"
+copy-button-tooltip-success: "Gekopieerd!"
+
+repo-action-links-edit: "Pagina bewerken"
+repo-action-links-source: "Broncode bekijken"
+repo-action-links-issue: "Een probleem melden"
+
+search-no-results-text: "Geen resultaten"
+search-matching-documents-text: "Gevonden documenten"
+search-copy-link-title: "Kopieer link om te zoeken"
+search-hide-matches-text: "Extra overeenkomsten verbergen"
+search-more-match-text: "meer overeenkomst in dit document"
+search-more-matches-text: "meer overeenkomsten in dit document"
+search-clear-button-title: "Wissen"
+search-detached-cancel-button-title: "Annuleren"
+search-submit-button-title: "Verzenden"
+
+crossref-fig-title: "Figuur"
+crossref-tbl-title: "Tabel"
+crossref-lst-title: "Listing"
+crossref-thm-title: "Stelling"
+crossref-lem-title: "Lemma"
+crossref-cor-title: "Conclusie"
+crossref-prp-title: "Voorstel"
+crossref-cnj-title: "Aanname"
+crossref-def-title: "Definitie"
+crossref-exm-title: "Voorbeeld"
+crossref-exr-title: "Oefening"
+crossref-ch-prefix: "Hoofdstuk"
+crossref-sec-prefix: "Paragraaf"
+crossref-eq-prefix: "Vergelijking"
+crossref-lof-title: "Lijst van figuren"
+crossref-lot-title: "Lijst van tabellen"
+crossref-lol-title: "Lijst van listings"
+
+environment-proof-title: "Bewijs"
+environment-remark-title: "Opmerking"
+environment-solution-title: "Oplossing"
+
+listing-page-order-by: "Sorteer op
```

Missing `"` the end of this line.

```suggestion
listing-page-order-by: "Sorteer op"
```